examples/plotting_with_folium.ipynb

```diff
@@ -0,0 +1,457 @@
+       "      <th>1</th>\n",
+       "      <td>2010</td>\n",
+       "      <td>Eyjafjallajokull</td>\n",
+       "      <td>Iceland</td>\n",
+       "      <td>63.630</td>\n",
+       "      <td>-19.620</td>\n",
+       "      <td>Stratovolcano</td>\n",
+       "      <td>POINT (-19.62 63.63)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2010</td>\n",
+       "      <td>Pacaya</td>\n",
+       "      <td>Guatemala</td>\n",
+       "      <td>14.381</td>\n",
+       "      <td>-90.601</td>\n",
+       "      <td>Complex volcano</td>\n",
+       "      <td>POINT (-90.601 14.381)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>2010</td>\n",
+       "      <td>Sarigan</td>\n",
+       "      <td>United States</td>\n",
+       "      <td>16.708</td>\n",
+       "      <td>145.780</td>\n",
+       "      <td>Stratovolcano</td>\n",
+       "      <td>POINT (145.78 16.708)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>2010</td>\n",
+       "      <td>Karangetang [Api Siau]</td>\n",
+       "      <td>Indonesia</td>\n",
+       "      <td>2.780</td>\n",
+       "      <td>125.480</td>\n",
+       "      <td>Stratovolcano</td>\n",
+       "      <td>POINT (125.48 2.78)</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Year                    Name        Country  Latitude  Longitude  \\\n",
+       "0  2010              Tungurahua        Ecuador    -1.467    -78.442   \n",
+       "1  2010        Eyjafjallajokull        Iceland    63.630    -19.620   \n",
+       "2  2010                  Pacaya      Guatemala    14.381    -90.601   \n",
+       "3  2010                 Sarigan  United States    16.708    145.780   \n",
+       "4  2010  Karangetang [Api Siau]      Indonesia     2.780    125.480   \n",
+       "\n",
+       "              Type                           geometry  \n",
+       "0    Stratovolcano  POINT (-78.44199999999999 -1.467)  \n",
+       "1    Stratovolcano               POINT (-19.62 63.63)  \n",
+       "2  Complex volcano             POINT (-90.601 14.381)  \n",
+       "3    Stratovolcano              POINT (145.78 16.708)  \n",
+       "4    Stratovolcano                POINT (125.48 2.78)  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "geometry = gpd.GeoSeries([Point(x,y) for x, y in zip(df['Longitude'], df['Latitude'])])\n",
+    "geo_df = gpd.GeoDataFrame(df[['Year','Name','Country', 'Latitude', 'Longitude', 'Type']], geometry=geometry)\n",
+    "\n",
+    "geo_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "world = gpd.read_file('zip://ne_110m_admin_0_countries.zip')"
```

Comment:
This dataset is included with geopandas, and can be read with `geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres`))`

(so you don't need to add the actual file in the PR)